Sprint-1/errors/1.js

```diff
@@ -1,4 +1,7 @@
 // trying to create an age variable and then reassign the value by 1
 
-const age = 33;
-age = age + 1;
+// const age = 33;
+// age = age + 1;
+
+var age =23;
+age =1;
```

Comment:
what is the difference between `var`, `const`, and `let`? Also, check the requirements (we want to increase the age variable by one)